src/babashka/fs.cljc

```diff
@@ -462,7 +466,16 @@
                                                        ^"[Ljava.nio.file.CopyOption;"
                                                        copy-options)
                                            :continue)
-                                         :continue)}))))
+                                         :continue)
+                           :post-visit-dir (fn [dir _ex]
+                                             (let [rel (relativize from dir)
+                                                   to-dir (path to rel)]
+                                               (if win?
```

Comment:
Since this problem doesn't exist on Windows, we can simply do nothing then, right?